docs/cli.md

```diff
@@ -101,6 +101,7 @@ The following flags are supported in the CLI:
 | `--alphabetize`           |       | `false`  | Sort types alphabetically                                                                                           |
 | `--array-length`          |       | `false`  | Generate tuples using array `minItems` / `maxItems`                                                                 |
 | `--default-non-nullable`  |       | `false`  | Treat schema objects with default values as non-nullable                                                            |
+| `--properties-required`   |       | `false`  | Treat schema objects without `required` as having all properties required.                                          |
```

Comment:
Minor change, and this is a little more verbose, but is a little more descriptive as to what it does. 

```suggestion
| `--properties-required-by-default`   |       | `false`  | Treat schema objects without `required` as having all properties required.                                          |
```